src/kirby/components/badge/badge.component.scss

```diff
@@ -25,12 +25,12 @@
   }
   
   &[colortype='alert'] {
-    --ion-color-base: #{get-color('kirby-alert')};
-    --ion-color-contrast: #{get-color('kirby-alert-contrast')};
+    --ion-color-base: #{get-color('kirby-warning')};
+    --ion-color-contrast: #{get-color('kirby-warning-contrast')};
   }
 
   &[colortype='danger'] {
     --ion-color-base: #{get-color('kirby-danger')};
-    --ion-color-contrast: #{get-color('kirby-white')};
+    --ion-color-contrast: #{get-color('kirby-contrast-light')};
```

Comment:
```suggestion
    --ion-color-contrast: #{get-color('kirby-danger-contrast')};
```

Reply:
Den her må du gerne vende med Tom - hvis den bliver kirby-danger-contrast så bliver den med sort tekst i stedet for hvid.